config/defaults.php

```diff
@@ -4,15 +4,15 @@
     'CrudView' => [
         'css' => [
             'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/css/bootstrap.css',
-            'CrudView./eonasdan-bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css',
+            'https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.14.30/css/bootstrap-datetimepicker.min.css',
```

Comment:
the downside to using the CDN for Bownty specifically is that we use vagrant, and can't always depend on internet - it's there 99% of the time though, so not a huge deal breaker